PowerUp/app/build.gradle

```diff
@@ -22,6 +21,7 @@ android {
             proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
         }
     }
+
```

Comment:
Blank line here can be removed

Reply:
it's fine. The last line should be blank.

Reply:
I did it intentionally as Murad suggested me in one of old PRs.